js-core/homeworks/phoneApp/src/main.js

```diff
@@ -0,0 +1,111 @@
+function PhoneApp() {
+    this.storage = [];
+}
+
+PhoneApp.prototype.checkNumber = function(numberToCheck) {
+    const toStringThePredictNumber = numberToCheck.toString();
+    const toArrayTheString = toStringThePredictNumber.split('');
+    let flag;
+    
+    toArrayTheString.forEach(elem => {
+        const toNumberElem = Number(elem);
+        if(isNaN(toNumberElem)) {
+            flag = false;
+        }
+    });
+    
+    if(flag === false) {
+        return flag;
+    } else {
+        return true;
+    }
+}
+
+PhoneApp.prototype.sortNumber = function(numberToSort) {
```

Comment:
you are not really sorting numbers, you are creating formatedPhoneNumber, the wrong name for that function is a bit dangerous